app/javascript/src/components/Dashboard/Contacts/index.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from "react";
+
+import { Button, Table, Pagination } from "neetoui";
+import { Container, Header } from "neetoui/layouts";
+
+import { CONTACT_DETAILS_DATA } from "./constant";
+import Menu from "./Menu";
+import { buildContactTableColumnData } from "./util";
```

Comment:
It should be `utils.js`. Refer: https://academy.bigbinary.com/learn-react/pages/naming-conventions-in-react#:~:text=Naming%20helper%20and%20validations%20files

Reply:
@yedhink I have renamed `util.jsx` to `utils.jsx` but when I tried to rename from `.jsx` to `.js`, It is showing me an error because I have some `jsx` code inside` utils.jsx` file.

Reply:
You can keep it as `utils.jsx`.